CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.10.2dev
 
+* [Fix] Fixed snippet behaviour with queries (#651, #652)
```

Comment:
changelog entries should be more detailed.

example: `Disabled CTE generation when detecting a %%sql cell with an existing CTE `